Automattic/sensei
# [Refactor] Generalize lesson block inserters

Some more preparation work for sharing helpers with the Quiz block

### Changes proposed in this Pull Request

* Break out `useAutoInserter` to automatically insert an empty inner block (as used for the module and lesson blocks)
* Break out Enter/Backspace logic to insert/remove/navigate siblings blocks into `useKeydownInserter`

### Testing instructions

* Edit a course with modules and lessons
* Make sure empty lessons are inserter to the end of the module when selected
* Make sure empty lesson can be deleted with `Backspace`, and new one created with `Enter`. (Should behave the same as before)
